drivers/adc/adc_stm32.c

```diff
@@ -502,15 +502,61 @@ static void adc_stm32_isr(const struct device *dev)
 	LOG_DBG("ISR triggered.");
 }
 
+static int adc_stm32_enable_and_wait_stabilisation(ADC_TypeDef *adc)
+{
+#if defined(CONFIG_SOC_SERIES_STM32L4X) || \
+	defined(CONFIG_SOC_SERIES_STM32L5X) || \
+	defined(CONFIG_SOC_SERIES_STM32WBX) || \
+	defined(CONFIG_SOC_SERIES_STM32G0X) || \
+	defined(CONFIG_SOC_SERIES_STM32G4X) || \
+	defined(CONFIG_SOC_SERIES_STM32H7X) || \
+	defined(CONFIG_SOC_SERIES_STM32WLX)
+	/*
+	 * Enabling ADC modules in L4, WB, G0 and G4 series may fail if they are
```

Comment:
Comment is not up to date with the list of SoCs available if the #if .
Also, would you be able to point to errata for other series  ?
(it could be done in the commit message as for L4, that would be perfect)